packages/dataviews/src/dataviews-layouts/table/index.tsx

```diff
@@ -278,6 +293,11 @@ function ViewTable< Item >( {
 			}
 		};
 
+	const isScrolledEnd = useIsScrolledEnd( {
+		scrollContainerRef: containerRef,
```

Comment:
The assumption here is that the scrollable container is the dataviews-wrapper div, is that always the case? Another possibility here could be to retrieve the parent scrollable of the "table".

Reply:
We discussed in https://github.com/Automattic/wp-calypso/pull/103005#discussion_r2071011447. This appears to be safe enough. Or do you prefer to pass the table’s ref and use [getScrollContainer](https://github.com/WordPress/gutenberg/blob/72c922b679cbc0d83220514165ea66837de131ca/packages/dom/src/dom/get-scroll-container.js#L15) to locate the scroll container? I’m not sure it’s worth introducing the @wordpress/dom package just for this.

Reply:
This works for me. Just a possibility I wanted to mention just in case.